project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -516,8 +516,8 @@ function c4m_user_og_c4m_leave_group_cta($group) {
     return;
   }
 
-  global $user;
-  if ($user->uid == $group->uid) {
+  // Group owner can't leave the group, needs to transfer ownership first.
+  if (c4m_og_user_is_owner('node', $group->nid)) {
```

Comment:
@DavidHernandez 
This one will check also administrators...  I think we only need to check ownership, hence the previous check was better ...